simpleocr/files.py

```diff
@@ -1,9 +1,9 @@
 import os
 import cv2
-from tesseract_utils import read_boxfile, write_boxfile
+from simpleocr.tesseract_utils import read_boxfile, write_boxfile
 
 IMAGE_EXTENSIONS = ['.png', '.tif', '.jpg', '.jpeg']
-DATA_DIRECTORY = 'data/'
+DATA_DIRECTORY = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
```

Comment:
I haven't had time to take a look at the API yet, but I have done some experimenting with my current packages and it appears that when using a `python setup.py sdist -r pypi` (source distribution to PyPI), using `__file__` is no problem at all. The code can be installed with `pip` and works fine. When using `python setup.py install` an egg is installed though, and the installation appears to fail on the data. I hope to be able to take a look Tuesday or Wednesday...